tests/test.js

```diff
@@ -56,43 +56,16 @@ describe("Embroider build", function () {
 });
 
 describe("When rootURL and publicPath do not match", function () {
-  let indexHtml, manifest, favicons, stylesheets;
+  let indexHtml, favicons, stylesheets;
 
   before(async function () {
     indexHtml = await getIndexHtml("rooturl-and-publicpath-differ");
     favicons = indexHtml.querySelectorAll(
```

Comment:
Are we still asserting against the favicons anywhere? Or should this be cleaned up as well?

Reply:
There were some path-related assertions, I removed them as the plugin would no longer touch favicons.

Reply:
Removed in https://github.com/jelhan/webpack-subresource-integrity-embroider/pull/16/commits/763ef9ea0a64e3af3f6411b5dbc98304816fc019 🙌